packages/ui-kit/src/helpers/getTimeZone.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Get the local time zone, falling back to "UTC" if unavailable.
+ */
+export function getTimeZone(): string {
+  let timeZone = 'UTC'
+  try {
+    timeZone = Intl.DateTimeFormat().resolvedOptions().timeZone
+  } catch (error) {
+    // Do nothing.
+  }
```

Comment:
Just curious, why is it necessary to add a try catch here? In what case it could break?

Reply:
```ts
Intl.DateTimeFormat().resolvedOptions()
```

is not defined in older browsers.

Additionally, `timeZone` could be undefined, so I should update for that case, too.